src/api/0x/index.ts

```diff
@@ -0,0 +1,136 @@
+import { OrderKind } from '@cowprotocol/contracts'
+import log from 'loglevel'
+import { SupportedChainId } from '../../constants/chains'
+import { Context } from '../../utils/context'
+import { getTokensFromMarket } from '../../utils/market'
+import { toErc20Address } from '../../utils/tokens'
+
+import BaseApi from '../base'
+
+import { Options, PriceQuoteParams } from '../cow/types'
+import ZeroXError, { logPrefix, ZeroXErrorResponse } from './error'
+import { ERC20BridgeSource, MatchaOptions, MatchaPriceQuote } from './types'
+
+// GPV2Settlement
+// https://etherscan.io/address/0x9008d19f58aabd9ed0d60971565aa8510560ab41
+const AFFILIATE_ADDRESS = '0x9008D19f58AAbD9eD0D60971565AA8510560ab41'
```

Comment:
why not referencing https://github.com/cowprotocol/cow-sdk/blob/base-api/src/constants/index.ts#L7 ?

Reply:
cool, didn't know it existed 👍